Dockerfile-slim.template

```diff
@@ -1,6 +1,7 @@
 FROM buildpack-deps:jessie-curl
 
-RUN groupadd -r node && useradd -r -g node node
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --create-home node
```

Comment:
Unless another regular user is added in the base image.
This way you will get an error and will know why it was not possible to get the ID 1000.